docs/content/dependencies.md

```diff
@@ -46,13 +45,12 @@ makes it easier to think about how the graph will be executed.
 
 ## Cross-Node References
 
-Resources may references one-another as long as the references do not introduce
-circular dependencies.  When creating a reference from one node to another we
-can use the `lookup` command to reference fields of an entry that are provided
-by that entries module.  The available fields will vary depending on the module
-and should be documented along with each module.  The example below illustrates
-using `lookup` to access fields from a `docker.image` node from within
-`docker.container`
+Resources may reference one another as long as the references do not introduce
+circular dependencies. When creating a reference from one node to another, we
+can use the `lookup` command to reference fields of another entry's module. The
+available fields will vary depending on the module and should be documented
+along with each module. The example below illustrates using `lookup` to access
```

Comment:
Also, how about "The example below uses `lookup` to get the right tag from a Docker image"?